config/spark.batch.conf.template

```diff
@@ -36,8 +36,8 @@ source {
 
 
 
-  # You can also use other input plugins, such as hdfs
-  # hdfs {
+  # You can also use other input plugins, such as file
+  # file {
```

Comment:
Why do you change this? Does other file system like OSS support?

Reply:
Firstly, there is no source named hdfs in current seatunnel-spark connector. And file source also support hdfs、aws 、oss protocol because it use hdfs client internally. Please correct me if I am wrong. thx